corehq/apps/reports/models.py

```diff
@@ -80,6 +81,17 @@ class HQUserType(object):
     included_defaults = (True, True, True, True, False)
 
     @classmethod
+    def user_type_mapping(cls):
```

Comment:
Nice Catch.. I added this for using UserTypeFilter but ended up relying on user_ids for each user type later on. Would remove this.

I am still slightly unclear about not using UserTypeFilter for user_types, exports used it but CaseList fetched ids for user types. We should definitely follow one thing for the entire application if possible?

Do we not save user_type for all forms and cases?